src/main/java/funda/prog101/Boiler.java

```diff
@@ -24,4 +24,12 @@ public void addCups() {
 			this.quantity++;
```

Comment:
this for is not needed, this can be just: this.quantity = this.limit;

>> I already added this comment to Oscar's pull request. Somebody nees to update this.